courses/udacity_intro_to_tensorflow_for_deep_learning/l08c03_moving_average.ipynb

```diff
@@ -113,7 +113,7 @@
       "source": [
         "import numpy as np\n",
         "import matplotlib.pyplot as plt\n",
-        "tf.enable_v2_behavior()\n",
+        "import tensorflow as tf\n",
```

Comment:
I don't remember the constraints for these course notebooks ... can we switch them to TF2 without an issue?

Reply:
if this question is to me: we do not use much in this particular notebook - just keras.metrics.
Unfortunately it runs things in eager fashion like 
```py
keras.metrics.mean_absolute_error(x_valid, naive_forecast).numpy()
```

Reply:
they are written in tf2 style, at least it looks like so far. So we can just check if version is 2.0

Reply:
Chatted with @MarkDaoust and this is a good change to make. Thanks!